library/smarty/plugins/function.headerTemplate.php

```diff
@@ -13,9 +13,10 @@
  * of the License, or (at your option) any later version.
```

Comment:
I may of asked, but what is library/core/src/render/BaseRenderer.php. And it will need a standard license header.